docs/self-managed/operational-guides/application-configs.md

```diff
@@ -0,0 +1,483 @@
+- `<componentName>.extraConfiguration`
+
+### componentName.configuration
+
+The `configuration` option is equivalent to an application configuration file. For example, `application.yaml`:
+
+```yaml
+operate:
+  configuration: |-
+    camunda.operate:
+      # ELS instance to store Operate data
+      elasticsearch:
+        # Cluster name
+        clusterName: elasticsearch
+        # Host
+        host: cpt-elasticsearch
+        # Transport port
+        port: 9200
+        numberOfShards: 3
+      # Zeebe instance
+      zeebe:
+        # Broker contact point
+        brokerContactPoint: "cpt-zeebe-gateway:26500"
+      # ELS instance to export Zeebe data to
+      zeebeElasticsearch:
+        # Cluster name
+        clusterName: elasticsearch
+        # Host
+        host: cpt-elasticsearch
+        # Transport port
+        port: 9200
+        # Index prefix, configured in Zeebe Elasticsearch exporter
+        prefix: zeebe-record
+    #Spring Boot Actuator endpoints to be exposed
+    management.endpoints.web.exposure.include: health,info,conditions,configprops,prometheus,loggers,usage-metrics,backups
+```
+
+### componentName.extraConfiguration
+
+The `extraConfiguration` option is used to supply extra configuration files.
+
+To use it, specify a key of the filename you want the option to have, where the value is the contents of that file. The most common use case for this would be supplying a `log4j2.xml` file. When the Helm chart reads this option, it mounts it on the current directory's `./config` folder:
+
+```xml
+operate:
+  extraConfiguration:
+    log4j2.xml: |-
+      <?xml version="1.0" encoding="UTF-8"?>
+      <Configuration status="WARN" monitorInterval="30">
+        <Properties>
+          <Property name="LOG_PATTERN">%clr{%d{yyyy-MM-dd HH:mm:ss.SSS}}{faint} %clr{%5p} %clr{${sys:PID}}{magenta} %clr{---}{faint} %clr{[%15.15t]}{faint} %clr{%-40.40c{1.}}{cyan} %clr{:}{faint} %m%n%xwEx</Property>
+          <Property name="log.stackdriver.serviceName">${env:OPERATE_LOG_STACKDRIVER_SERVICENAME:-operate}</Property>
+          <Property name="log.stackdriver.serviceVersion">${env:OPERATE_LOG_STACKDRIVER_SERVICEVERSION:-}</Property>
+        </Properties>
+        <Appenders>
+          <Console name="Console" target="SYSTEM_OUT" follow="true">
+            <PatternLayout pattern="${LOG_PATTERN}"/>
+          </Console>
+          <Console name="Stackdriver" target="SYSTEM_OUT" follow="true">
+            <StackdriverLayout serviceName="${log.stackdriver.serviceName}"
+              serviceVersion="${log.stackdriver.serviceVersion}" />
+          </Console>
+        </Appenders>
+        <Loggers>
+          <Logger name="io.camunda.operate" level="debug" />
+          <Root level="debug">
+            <AppenderRef ref="${env:OPERATE_LOG_APPENDER:-Console}"/>
+          </Root>
+        </Loggers>
+      </Configuration>
+```
+
+## How to get the application.yaml properties for a particular component to override
```

Comment:
This is a `H2` title and will show on the right side table of content. It should be much shorter (the same for other H2 titles here too).

Reply:
I changed the H2 titles to be shorter. I think one thing to be wary of with shorter titles is that it may be harder to find it in the search bar.